assets/js/components/ClusterDetails/AscsErsClusterDetails.jsx

```diff
@@ -65,13 +70,19 @@ const nodeDetailsConfig = {
 };
 
 function AscsErsClusterDetails({
+  clusterID,
   clusterName,
+  selectedChecks,
+  hasSelectedChecks,
   cibLastWritten,
   provider,
   hosts,
   sapSystems,
   details,
   catalog,
+  lastExecution,
+  onStartExecution,
```

Comment:
Should we provide a default noop function to `onStartExecution` and `navigate`?
```js
onStartExecution = () => {},
navigate = () => {},
```
We do that in the HanaClusterDetails component.
What's the way we prefer?